index.js

```diff
@@ -1,6 +1,11 @@
 import Twitter from 'twit';
+import Promise from 'pinkie-promise';
```

Comment:
what do you think about including `babel-runtime` to get this for free? as far as Promise support landed in ECMAScript and NodeJS we can semi-rely on this construct, but still polyfill that in case of nodejs versions lower than v4?